packages/components/src/ClickableStyleWrapper/ClickableStyleWrapper.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { Story } from "@storybook/react/types-6-0";
+import React from "react";
+import ClickableStyleWrapper from "./ClickableStyleWrapper";
+
+export default {
+  title: "ClickableStyleWrapper",
+  component: ClickableStyleWrapper,
+};
+
+type Args = React.ComponentProps<typeof ClickableStyleWrapper>;
+
+const Template: Story<Args> = (args) => <ClickableStyleWrapper {...args} />;
+
+export const ClickableStyleWrapperAsAnATag = Template.bind(null);
+ClickableStyleWrapperAsAnATag.args = {
```

Comment:
I'm not sure it's useful to surface visual stories, since this is now more of an implementation detail 🤔  Instead we could make this `.stories.mdx` so it's more of a documentation page on usage. MDX files can still render code  to show examples and such

Reply:
I feel like it's still useful to have this component in storybook so when devs go to use it with react router (or whatever), they can see how it works and play around with it.

I'll make a ticket for taking another look at the button, link, and clickable stories generally to figure out what we think is the best way to present this set of components based on who's using them and how.

Reply:
Feel free to add more suggestions and clarify the ones I listed so far: https://app.clubhouse.io/czi-edu/story/160018/reorganize-button-link-and-clickablestyle-stories